documentation/staging/content/faq/external-clients.md

```diff
@@ -366,6 +381,129 @@ To enable an 'unknown host' source WebLogic Server to initiate EJB, JMS, or JTA
+```
+Here is a snippet of WDT model YAML file configuration for enabling server affinity in a cluster 'cluster-1':
+
+```yaml
+topology:
+    Cluster:
+        'cluster-1':
+            DynamicServers:
+                ServerTemplate:  'cluster-1-template'
+                ServerNamePrefix: 'managed-server'
+                DynamicClusterSize: '5'
+                MaxDynamicClusterSize: '5'
+                MinDynamicClusterSize: '0'
+                CalculatedListenPorts: false
+            DefaultLoadAlgorithm: 'round-robin-affinity'
+```
+
+#### Configuring External Listen Address for WebLogic Default Channel
+
+WebLogic Server provides an External Listen Address/DNS Name for use in RMI stubs to allow external clients to connect to the server through a Kubernetes external service name. For more information on 'External Listen Address' (i.e. ExternalDNSName attribute) on the ServerMBean, see ExternalDNSName attribute in [ ServerMBean reference ](https://docs.oracle.com/en/middleware/fusion-middleware/weblogic-server/12.2.1.4/wlmbr/core/index.html).
+
+Here are steps for setting the 'ExternalDNSName' attribute on the ServerMBean, for the default channel, for a domain source type __Domain in PV__:
+
+```
+How to set the 'External Listen Address' on the ServerMBean for stand-alone server 'AdminServer': 
+=========================================== 
+
+1. Go to Admin console 
+2. Expand the Environment link on the left panel 
+3. Click on the Servers link on the sub-panel
+4. On the right panel, click on AdminServer -> Configuration -> General tabs. 
+5. Click on the Advanced link to expand the server configuration page.
+6. Enter the service name for the WLS server pod in the External Listen Address field.
+   For example:
+     i. For Admin Server the service name would be of the form <DOMAIN_UID>-admin-server.<NAMESPACE> 
+     ii. For a managed server the service name would be of the form <DOMAIN_UID>-managed-server${id}.<NAMESPACE>
+``` 
+
+Here are steps for setting the 'ExternalDNSName' attribute on the ServerTemplateMBean, for the default channel, for a domain source type __Domain in PV__:
+
+```
+How to set the 'External Listen Address' on the ServerTemplateMBean for server template named 'cluster-1-template': 
+=========================================== 
+
+1. Go to Admin console 
+2. Expand the Environment link on the left panel 
+3. Click on the Clusters link on the sub-panel
+4. Click on the Server Templates link on the sub-panel
+5. On the right panel, click on cluster-1-template -> Configuration -> General tabs. 
+6. Click on the Advanced link to expand the server configuration page.
+7. Enter the service name for the WLS server pod in the External Listen Address field.
+   For example:
+     i. For Admin Server the service name would be of the form <DOMAIN_UID>-admin-server.<NAMESPACE> 
+     ii. For a managed server the service name would be of the form <DOMAIN_UID>-managed-server${id}.<NAMESPACE>
+``` 
```

Comment:
In WKO environment any change to config.xml is lost if we do not use the Domain-on-PV model. So this is not a suggested way to modify the cluster attribute.